src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -31,24 +53,45 @@ internal override ContractTask Initialize(ApplicationEngine engine)
             return ContractTask.CompletedTask;
         }
 
+        /// <summary>
+        /// Gets the network fee per byte of transactions.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <returns>The network fee per byte of transactions.</returns>
         [ContractMethod(CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
         public long GetFeePerByte(DataCache snapshot)
         {
             return (long)(BigInteger)snapshot[CreateStorageKey(Prefix_FeePerByte)];
         }
 
+        /// <summary>
+        /// Gets the execution fee factor.
```

Comment:
Maybe explain more. "Gets the execution fee factor. This is a multiplier that can be adjusted for raising all fees in a linear manner/fashion."

Reply:
Updated.